packages/pwa-kit-react-sdk/src/ssr/universal/components/route-component/index.js

```diff
@@ -406,7 +407,12 @@ export const getRoutes = (locals) => {
     if (typeof routes === 'function') {
         _routes = routes()
     }
-    const allRoutes = [..._routes, {path: '*', component: Throw404}]
+    const allRoutes = [
+        // NOTE: this storefront preview route needs to be above _routes, in case _routes has a fallback route of `path: '*'`
+        {path: '/__storefront-preview/refetch-data', component: RefetchData},
```

Comment:
I agree with the above comment. This should be agnostic to storefront preview. Can't we call it something simple like `__refresh`, `__force_reload` .. something along those lines. 

We should also make it apparent that this is intended to be a private endpoint and developers should never rely on it being available since we might want to change it in the future. Because we know it would be a better experience if the data simply updated without having to navigate away from the current page and back.

Reply:
We decided on `__mrt/refresh`.